amundsen_application/static/config/config-default.ts

```diff
@@ -15,6 +15,13 @@ const configDefault: AppConfig = {
     key: 'default-key',
     sampleRate: 100,
   },
+  lineage: {
```

Comment:
it would be good if we could add a comment for what the config is used for. In fact, it would be good to have a comment for all the config here(could be a different pr)

Reply:
I'll add some comments to the type definitions in config.types.ts.